### PR TITLE
New post community meeting

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**Changes**
- Adding in a new News Post
- Small fix to remove unwanted output on screen

image

image

### CODE REVIEW

File: _posts/2018-06-12-first-community-meeting.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: news
+section: news
+title: Announcing our Community Meeting!
+author: laurafitzgerald
+---
+
+We are delighted to announce that we held our first AeroGear Community meeting on Monday 11th June 2018. 
+Thanks to everyone who was able to attend. For those who couldn't, you can find more detail(including the recording, discussions had and decisions made) on [this thread](https://groups.google.com/forum/#!topic/aerogear/ohcRq2D6r2Q){:target="_blank"}.
```
Comment (dimitraz) [inline _posts/2018-06-12-first-community-meeting.md]: Very small nit: space after `detail`

Comment (laurafitzgerald) [reply _posts/2018-06-12-first-community-meeting.md]: :+1 done :)

**Comment by laurafitzgerald** [comment]
@jasonmadigan I've made the change to remove the old posts. Right now there is only the new post that I've added but we could ask people to start contributing here between now and the release and then we would have some nice new shiny content in there. wdyt?

**Comment by dimitraz** [comment]
@laurafitzgerald Since the old posts are gone now, maybe you can delete the authors from people.yaml? I don't think they're used elsewhere but I could be mistaken

**Comment by laurafitzgerald** [comment]
@dimitraz yes I can do. does it make sense to leave the ones in place for those that are still on the team?

**Comment by jasonmadigan** [comment]
thanks for the work here @laurafitzgerald - could you add a quick screenshot as well? will checkout locally too

**Comment by jasonmadigan** [comment]
seeing one older post still show up: 

screen shot 2018-06-13 at 14 10 56

**Comment by jasonmadigan** [comment]
not strictly part of this PR, but there's a typo on the heading for the page "Artciles" -> "Articles"

**Comment by laurafitzgerald** [comment]
@jasonmadigan I've removed a section that searches for blogs. I believe it comes through the users github url from people.yml which is where that last new entry item was coming from. Will add a screen shot now to the description. Would you mind verifying that this last change works on your end.

**Comment by laurafitzgerald** [comment]
@jasonmadigan also updating the icons to match those used in @dimitraz [pr](https://github.com/aerogear/aerogear.org/pull/733). The last commit with styling contains the same styling from that pr so should disappear once rebased.

**Comment by jasonmadigan** [comment]
looks good @laurafitzgerald!

**Pull request merged**